README.md

```diff
@@ -192,6 +194,33 @@ us to build a hierarchical dependency tree of all puzzles, like
 for example. Technically, of course, you can abuse the system
 and put a dummy `#1` marker everywhere.
 
+### Multiline examples
+
+```xml
+<!-- 
+ ~ if comment should be started and closed by special symbols, then place them in
+ ~ a separate lines without any text
+ ~ And any symbols can be used as a prefix, it will be excluded from the text
+ ~
+ ~ @todo #34 Description can not be created in one line with comment
+ ~ symbols. Just use at least the same amount of the spaces, as on the first line. 
+-->
+```
+
+```java
+/**
+ * @todo #36 Multiline text can use the same prefix in all lines or the same 
+ *  amount of spaces.
+    So this will be added to the puzzle description. If you want to divide the 
+ *  puzzle by empty line, just add a space to that line  
```

Comment:
IMHO we should reconsider and improve this rule. Using a trailing space for making decisions isn't good.
First, it's non-obvious.
Second, it's invisible, so it's really hard to catch such issues on code review.
Third, users may want to run auto-formatting tools, that can remove such "noise". In this case, users will face a massive close and creation of the issues.

How about using a backslash? Like this:

```java
/**
 * @todo #36 Multiline text can use the same prefix in all lines or the same 
 *  amount of spaces.
    So this will be added to the puzzle description. If you want to divide the 
 *  puzzle by empty line, just add a backslash to that line
 *  \
 *  and continue the text after.
 *
 *  This line is not part of the puzzle, because the line before contains less
 *  spaces then the second line.
 */
```

@yegor256 WDYT?

Reply:
@pnatashap @php-coder indeed, a "blind" space at the end of a line is provocation of a mistake

Reply:
Good symbol should be clear enough for developers. 
Also will check if it works for second line as described in https://github.com/cqfn/pdd/issues/163